hardhat.config.ts

```diff
@@ -149,30 +149,15 @@ const config: HardhatUserConfig = {
   solidity: {
     compilers: [
       {
-        version: "0.8.13",
-        settings: {
-          optimizer: {
-            enabled: true,
-            details: {
-              yul: !process.env.CI,
-            },
-          },
-          outputSelection: {
-            "*": {
-              "*": ["storageLayout"],
-            },
-          },
-        },
-      },
-      {
-        version: "0.8.20",
+        version: "0.8.25",
         settings: {
           optimizer: {
             enabled: true,
             details: {
               yul: !process.env.CI,
             },
           },
+          evmVersion: "paris",
```

Comment:
`paris` didn't include `PUSH0`

```suggestion
          evmVersion: "shanghai",
```

Reply:
I think Shanghai EVM version `PUSH0` opcode is not supported by every chain.

Reply:
I didn't do my own research, but AFAIK all chains we use already support `PUSH0`. I may be wrong, though. @GitGuru7 @0xlucian do you guys know any particular chain from our list that doesn't support `PUSH0`?

Reply:
Looks like opBNB mainnet and testnet do not support shanghai yet. They plan to add support for it in a mid-June release. I'd merge the `paris` version for now, unless we can specify evm version per chain (I haven't found the info on how to do that though – looks like compilation step is chain-agnostic in hardhat).